backend/backend.go

```diff
@@ -211,7 +211,7 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 	backend.notifier = notifier
 	backend.socksProxy = socksproxy.NewSocksProxy(
 		backend.config.AppConfig().Backend.Proxy.UseProxy,
-		backend.config.AppConfig().Backend.Proxy.ProxyAddressOrDefault(),
+		backend.config.AppConfig().Backend.Proxy.ProxyAddress,
```

Comment:
What if someone's app config.json is corrupted or they modified it manually? For me ideally the app would be more resilient and notify the user it couldn't set the proxy address at startup in that case.

Reply:
I agree, this is about not panicking at startup. I really don't know at the moment how to do that, and would leave it for another day.

Reply:
Here's a quick idea for the future: collect all errors on start up and present them to the user in some form as soon as the app appears on the screen. In an extreme case where nothing can be shown, run a system native alert  or notification?

In the case of socksproxy, it would retain the error and switch it off. A notification would come up: "Tor proxy has been turned off due to a configuration error: ..." Or don't turn it off and display: "The app is offline due to Tor proxy configuration error: ..."